docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # -- build-environment --
 # see https://docs.docker.com/engine/userguide/eng-image/multistage-build/
 
-FROM quay.io/travelping/alpine-erlang:23.3.4 AS build-env
+FROM quay.io/travelping/alpine-erlang:23.3.4.12 AS build-env
```

Comment:
Latest version of 23 is `23.3.4.13`. What the reason of using version `23.3.4.12`?

Reply:
2.3.4.13 is not available at https://quay.io/repository/travelping/alpine-erlang?tab=tags.

Reply:
Hm... looks like need to update. But in any case, maybe better to use just `23.3` - this will always use latest version with including minor versions.

Reply:
and i dislike that because history has shown that $people change the underlaying alpine version from one erlang-patch-version to the next one and as a result, the built image were broken in subtle ways coz the musl library in the now subtle wrong base image was not the one on which erlang was built.

so, for 23.3.4.12 i _do_ know that alpine:3.15 is used that is the combination i have "locked".